### DISCUSSION TITLE
Sticky header

### DISCUSSION THREAD

**Discussion by jjc704** [laravel/nova-issues#5806] [Ideas & Feature Requests] [answered]
It would be helpful to have sticky header in the resource index view

**Comment by davidhemphill** [comment]
Which part of the header specifically? Can you upload an image?

**Comment by FY0u11** [comment, marked as answer]
If you say about the part with columns name, then I did it sticky applying some css styles to the head tag:

![image](https://github.com/laravel/nova-issues/assets/48677295/9b7dbd56-4854-4ffc-884d-0bdf75481f19)

But, to make it work, you also need remove overflows from the tag about:

![image](https://github.com/laravel/nova-issues/assets/48677295/00fe6b19-fa34-4942-b683-518ce029c9af)
